app/components/x-list-cell.js

```diff
@@ -104,6 +81,32 @@ export default Component.extend({
    */
   highlight: false,
 
+  /**
+   * The list of possible columns
+   * @property columns
+   * @type {Array<{id: String, name: String, maxWidth: Number, width: Number}>}
+   */
+  columns: computed(() => []),
```

Comment:
Can we just set `columns` in `init`? I tend to prefer to set properties on `init` rather than having computeds that are just intended to get around having an array as a default prop.